tests/kafkatest/services/console_consumer.py

```diff
@@ -59,7 +59,8 @@ class ConsoleConsumer(KafkaPathResolverMixin, JmxMixin, BackgroundThreadService)
 
     def __init__(self, context, num_nodes, kafka, topic, group_id="test-consumer-group", new_consumer=True,
                  message_validator=None, from_beginning=True, consumer_timeout_ms=None, version=DEV_BRANCH,
-                 client_id="console-consumer", print_key=False, jmx_object_names=None, jmx_attributes=None,
+                 client_id="console-consumer", print_key=False,
+                 jmx_enabled=True, jmx_object_names=None, jmx_attributes=None,
```

Comment:
Can we infer `jmx_enabled` based on `jmx_object_names` and `jmx_attributes`? Are there any cases we would want the jmx tool if these are both `None`?

Reply:
looks like we already refuse to start if not objects are specified

```python
    def start_jmx_tool(self, idx, node):
        if self.jmx_object_names is None:
            self.logger.debug("%s: Not starting jmx tool because no jmx objects are defined" % node.account)
            return
        ...
```

so sounds like we can just say
```python
    self.jmx_enabled = False if self.self.jmx_object_names is None else True
```